### ISSUE TITLE
Accessibility: Window Menu items should have accessible name labeling reflecting the name of the menu (E.g. File).

### ISSUE THREAD

**Issue by auchenberg** [Microsoft/vscode#51492]
Learnings from NVAccess testing:

When pressing the shortcut, focus first goes to the menu itself. However, this menu does not have a label. Thus NVDA just says "menu sub menu". It would be good if it had an accessible name reflecting the name of the menu (E.g. File).

**Comment by sbatten** [comment]
duplicate of #17224
